lib/src/Pos/Util/Mockable.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE PolyKinds    #-}
```

Comment:
This is being added to the `lib` package. Is there a reason its being added here rather than further down the stack? Currently `lib` seems to be a bit of dumping ground for random bits of code, much of which could reasonably be put elsewhere.